### PR TITLE
Upgrade to ocamlformat.0.22.4

### PR DESCRIPTION
**Pull request by NathanReb**

### CODE REVIEW

**Review by Leonidas-from-XIV — APPROVED** [review]
The changes look ok to me, not much wild stuff going on and in these cases I actually prefer the new formatting.

**Comment by NathanReb** [comment]
Unfortunately, the ocaml-ci bug strikes again and we can't merge this. I've pinged the maintainer on my old PR and we're trying to find a durable solution!

**Comment by NathanReb** [comment]
I'm closing for now! I realize that we could start with a 0.21 upgrade first so I'll give this a spin!

**Pull request closed without merging**
